src/fragments/lib/push-notifications/android/getting_started/10_pre_req.mdx

```diff
@@ -0,0 +1,5 @@
+* An Android application targeting at least Android SDK API level 24 with Amplify libraries integrated
+    * For a full example of creating Android project, please follow the [project setup walkthrough](/lib/project-setup/create-application)
+
+
+Push Notifications are delivered via Firebase Cloud Messaging (FCM), In order to use FCM, you need to register your app on the Firebase console. See [Setting up push notification services](/lib/push-notifications/setup-push-service/) for more information.
```

Comment:
I think this sentence is confusing. I would either remove this sentence or explain all we need is a server key and id from the Firebase project.

Reply:
Also, in my opinion, this section should not be at the very bottom on the sidebar. Because this is the natural next step to take to enable the capabilities. I would even argue to have it in the same page since it is part of the getting started guide. @abdallahshaban557 what do you think?

Reply:
We had them in separate pages because of how long the getting started page would be otherwise. We can see customer feedback afterwards and figure out whether it should be in the same page or not.

Reply:
But we have the headers because of this reason, right? I disagree with you on this one but let's talk to customers afterwards like you have mentioned to make an action.